docs/balloon.md

```diff
@@ -0,0 +1,74 @@
+# Balloon
+
+Cloud Hypervisor emulates a balloon device based on the VIRTIO specification.
+Its main purpose is to provide the host a way to reclaim memory by controlling
+the amount of memory visible to the guest. But it also provides some interesting
+features related to guest memory management.
+
+## Parameters
+
+`BalloonConfig` (known as `--balloon` from the CLI perspective) contains the
+list of parameters available for the balloon device.
+
+```rust
+struct BalloonConfig {
+    pub size: u64,
+    pub deflate_on_oom: bool,
+    pub free_page_reporting: bool,
+}
+```
+
+```
+--balloon <balloon>	Balloon parameters "size=<balloon_size>,deflate_on_oom=on|off,free_page_reporting=on|off"
+```
+
+### `size`
+
+Size of the balloon device. It is substracted from the VM's total size. For
+instance, if creating a VM with 4GiB of RAM, along with a balloon of 1GiB, the
+guest will find 3GiB of accessible memory.

```

Comment:
s/find 3GiB/will be able to use 3GiB/

The guest sees all the RAM an unless it is balloon enlightened is entitled to all of it.